specifications/sep-payment-channel-mechanism.md

```diff
@@ -0,0 +1,631 @@
+payment channel has a single value for m it is computable as, s+(m*e).
+
+### Processes
+
+#### Setup
+
+To setup the payment channel:
+
+1. I creates the escrow account E.
+2. R creates the reserve account V.
+3. Set variable initial states:
+   - s to E's sequence number + 1.
+   - i to 0.
+   - e to 0.
+5. I and R build the formation transaction F.
+6. Increment i.
+7. Sign and exchange a closing transaction C_i, that closes the channel with
+disbursements matching the initial contributions.
+8. Sign and exchange a declaration transaction D_i.
+9. I and R sign and exchange signatures for formation transaction F.
+10. I or R submit F.
+
+The transactions are constructed as follows:
+
+- F, the _formation transaction_, deposits I and R's contributions to escrow
+account E, R's reserves to reserve account V, and changes escrow account E and
+reserve account V to be 2-of-2 multisig accounts. F has source account E, and
+sequence number set to s_i.
+
+  F contains operations:
+
+  - Operations sponsored by I:
+    - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies participant
+    I as a sponsor of future reserves.
+    - One or more `SET_OPTIONS` operations adjusting escrow account E's
+    thresholds such that I and R's signers must both sign, and adding I's
+    signers to E.
+    - One or more `SET_OPTIONS` operations adding I's signers to V.
+    - One or more `CHANGE_TRUST` operations adding trustlines to E.
+    - One `END_SPONSORING_FUTURE_RESERVES` operation that stops I sponsoring
+    future reserves of subsequent operations.
+  - Operations sponsored by R:
+    - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies reserve
+    account R as a sponsor of future reserves.
+    - One or more `SET_OPTIONS` operations adjusting escrow account V's
+    thresholds such that R and I's signers must both sign, and adding R's
+    signers to V.
+    - One or more `SET_OPTIONS` operations adding R's signers to E.
+    - One `END_SPONSORING_FUTURE_RESERVES` operation that stops R sponsoring
+    future reserves of subsequent operations.
+  - One or more `PAYMENT` operations depositing I's contribution to E.
+  - One or more `PAYMENT` operations depositing R's contribution to E.
+  - One or more `PAYMENT` operations depositing R's reserves to V, for each
+  trustline on E that will be used to sponsor claimable balances at
+  disbursement.
+  
+- C_i, see [Update](#Update) process.
+
+- D_i, see [Update](#Update) process.
+
+#### Update
+
+To update the payment channel state, the participants:
+
+1. Increment i.
+2. Sign and exchange a closing transaction C_i.
+3. Sign and exchange a declaration transaction D_i.
+
+The transactions are constructed as follows:
+
+- C_i, the _closing transaction_, disburses funds to R and changes the signing
+weights on E such that I unilaterally controls E.  C_i has source account E,
+sequence number s_i+1, a `minSeqAge` of O (the observation period time
```

Comment:
Wouldn't this be `s_i+m`?

Reply:
The C tx needs to be the second tx in the tx set. s_i is the seq of the first tx in the tx set, that's why C is s_i+1.